src/domain/auth/components/login/Login.tsx

```diff
@@ -42,6 +42,10 @@ function Login(props: Props) {
 
   const isAuthenticated = useSelector(isAuthenticatedSelector);
 
+  if (Boolean(props.auth.error)) {
+    toastNotification({ onClose: () => props.resetError() });
+  }
```

Comment:
This solution seems to be working, only one notification is shown at the time. I'm still thinking if I should wrap this with `useEffect` to be sure.

Reply:
Well, you are correct. The render function should not have side effects unless wrapped in a `useEffect`. But I think that an even better route could be to set the toast in the authentication routine itself, in which case you would not need to use a `useEffect` either. Now the firing of the notification doesn't need to be tied to this component anymore.